.github/workflows/onPr.yaml

```diff
@@ -0,0 +1,11 @@
+name: "Run Jobs on PR"
+on: pull_request
+
+jobs:
+  execute:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: jpmcb/prow-github-actions@v1.1.3
+        with:
+          jobs: 'lgtm'
+          github-token: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
I am not sure how comfortable we are with giving write level token access to this specific action. I believe the original intent of the issue was to just support PR labeling which is a much more restrictive action and requires fewer permissions. This functionality is also offered by github by default now using the auto merge feature. I think it might be better to leave this feature turned off.